justfile

```diff
@@ -204,7 +204,7 @@ wipe: wipe-docker wipe-coordinator wipe-maker wipe-app wipe-webapp
 wipe-docker:
     #!/usr/bin/env bash
     set -euxo pipefail
-    docker-compose down -v
+    docker compose down -v
```

Comment:
I fear this will break it for me since I did not install compose as plugin but as separate binary.

Reply:
Hmm, on the other hand, I do not have compose and only have the plugin. According to the site the separate binary is not recommended:
![image](https://github.com/get10101/10101/assets/6688948/cf4d8337-586f-41f1-ae64-d5e1606fb912)

Reply:
I can drop this commit if we want but it'd be good to find a solution that works for both of us

Reply:
Happy to change my setup if it's fine for everybody else @bonomat @luckysori

Reply:
no hard feelings, on my machine both seems to work :)

Reply:
I'm okay with this change.